desc/compute/_profiles.py

```diff
@@ -478,7 +478,8 @@ def _gradp_mag_vol(params, transforms, profiles, data, **kwargs):
     transforms={"grid": []},
     profiles=["iota", "current"],
     coordinates="r",
-    data=["psi_r", "lambda_t", "lambda_z", "g_tt", "g_tz", "sqrt(g)"],
+    data=["psi_r", "iota_0_num", "iota_0_den"],
```

Comment:
maybe I misunderstand the dependency logic, but isn't the iota_0_den and iota_0_num axis limit data? not data?

Reply:
oh nvm I see, `iota_0_num` is what you call the zero current iota. That name could be a bit confusing when we are talking about axis limits... but I can't think of a better name for it right now. Maybe `iota_zero_curr` or `iota_vac`?